flutter_app/lib/mavlink_communication.dart

```diff
@@ -95,11 +93,15 @@ class MavlinkCommunication {
         _pitchSpeedController.add(attitude.pitchspeed);
         _yawSpeedController.add(attitude.yawspeed);
         _timeBootMsPitchController.add(attitude.timeBootMs);
+      } else if (frame.message is GlobalPositionInt) {
```

Comment:
Change conditional to use switch case.

Reply:
Can't implement this behavior with a switch statement since these are not constants. They are subclasses.

Reply:
Unfortunate.